src/domains/tasks/ViewTasks/components/tests/OffersTable.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { screen } from "@testing-library/react";
+import { OffersTable } from "../TasksPage";
+
+import { renderWithPolarisTestProvider } from "src/test/utils";
+import { ComponentProps } from "react";
+
+function renderOffersTable(props: ComponentProps<typeof OffersTable>) {
+  return renderWithPolarisTestProvider(<OffersTable {...props} />);
+}
+
+function setupWithOneOffer() {
```

Comment:
Got the idea from Kent C. Dodds

https://kentcdodds.com/blog/avoid-nesting-when-youre-testing